index.html

```diff
@@ -12,6 +12,9 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Crafting reference for One Hour One Life</title>
+    <link rel="icon" type="image/png" href="./src/assets/favicons/favicon-16x16.png" sizes="16x16">
+    <link rel="icon" type="image/png" href="./src/assets/favicons/favicon-32x32.png" sizes="32x32">
+    <link rel="icon" type="image/png" href="./src/assets/favicons/favicon-96x96.png" sizes="96x96">
```

Comment:
It is best to not use `./src` since those files are dynamically processed. Also, the build script will not copy it to `./dist` because they are not referenced in the JS.

I'm not familiar enough with webpack structure. @Kazetsukai, should these images go in `static`? It gets a little messy with having `static-dev` too.

Reply:
I'm a fan of treating the favicon as similar to the other "hardcoded" images we have (the [https://github.com/Kazetsukai/onetech/blob/master/src/components/ObjectImage.vue#L78](hand) and stopwatch for example)

That would mean referring to it from `../assets/favicons`, and yeah not using `./src`. I'm just not 100% sure whether the current webpack setup will pick those up and process, but if `../assets/favicons` works in dev it should also work for the build.